Restrict plugin to module generated links

korontari opened the pull request:
Restrict the Javascript plugin to processing module generated links only and ignore other (manually) created ones.

korontari committed:
Restrict plugin to module generated links

Restrict the Javascript plugin to processing module generated links only and ignore other (manually) created ones.

Rudloff commented on glightbox.module:
```diff
@@ -56,6 +56,17 @@ function glightbox_menu() {
+  // This is done for security reasons so that only the links generated
+  // by this module are processed by the javascript plugin. Manually 
+  // injected links are thus ignored.
+  config_set('glightbox.settings', 'selector', 'glightbox-' . backdrop_random_key(8));
```
This seems expensive and would slow down every request.
I think it would make more sense to generate this in theme_glightbox() and not store it in config.

korontari replied:
Yea, I see. I've made a new commit where I moved the selector generation to the theme registering function.

korontari committed:
Removed the lightbox selector from the config

The unique lightbox selector is now generated in the theme register function(glightbox_theme) and passed on via variables to the theme function(theme_glightbox).

korontari committed:
fix typo

korontari committed:
library update

javascript library updated to v3.3.1

korontari committed:
Fix #7
